__fixtures__/test-project/package.json

```diff
@@ -20,6 +20,5 @@
   },
   "prisma": {
     "seed": "yarn rw exec seed"
-  },
-  "packageManager": "yarn@3.2.0"
```

Comment:
This may have been accidentally committed; it contradicts the value in `.yarnrc.yml`